components/Animations/samples/Animations.Samples.csproj

```
@@ -5,6 +5,11 @@
 
   <!-- Sets this up as a toolkit component's sample project -->
   <Import Project="$(ToolingDirectory)\ToolkitComponent.SampleProject.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\Extensions\src\CommunityToolkit.WinUI.Extensions.csproj"/>
```

With non-sdk style projects in the solution, we can't always be sure that relative paths will resolve from where we expect them to. Let's use the absolute path instead, predefined in `Directory.Build.props`:
```suggestion
    <ProjectReference Include="$(ToolkitExtensionSourceProject)"/>
```